### ISSUE TITLE
Active Directory integration

### ISSUE THREAD

**Issue by IlyaFinkelshteyn** [appveyor/ci#2567]
There are couple of options:
- `Azure AD` login button
-  Authenticate AppVeyor (email-based) users against domain controller or Azure AD (set at Account level settings)
-  Authenticate AppVeyor (email-based) users against domain controller (for AppVeyor Enterprise)
- SAML integration

**Comment by UziTech** [comment]
Is on premises Active Directory going to be supported in AppVeyor?

**Comment by FeodorFitsner** [comment]
@UziTech you mean to support it in on-premise AppVeyor Server?

**Comment by UziTech** [comment]
Yes support on premises Active Directory in on premises AppVeyor
